---
title: "paper_draft"
author: "DM"
format: pdf
editor: visual
bibliography: refs.bib
execute: 
  echo: false 
---

```{r echo=FALSE, message=FALSE}
source('_targets_packages.R')
library(targets)
library(knitr)
library(FactoMineR)
library(factoextra)
library(survival)
library(clValid)

source('R/utils.R')

theme_set(theme_bw())
```

## Title

-   the name of the organism studied

-   the particular aspect or system studied

-   the variable(s) manipulated.

## Abstract

passive voice

(typically 150-300 words)

-   A brief introduction to the topic that you're investigating.

-   Explanation of why the topic is important in your field/s.

-   Statement about what the gap is in the research.

-   Your research question/s / aim/s.

-   An indication of your research methods and approach.

-   Your key message.

-   A summary of your key findings.

-   An explanation of why your findings and key message contribute to the field/s

## Introduction

Neurodevelopment disorders are a diverse and heterogeneous group of conditions impacting the development of the nervous system, brain function, physical development, emotional development and learning ability. A recently identified condition known as Neurodevelopmental Disorder with Microcephaly, Arthrogryposis, and Structural Brain Anomalies (NEDMABA) (MIM:[618622](https://omim.org/entry/618622)) has been described in children with bi-allelic loss-of-function variants in *SMPD4* [@magini2019loss]. Sphingomyelinases such as *SMPD4* play an important celluar role by hydrolyzing sphingomyelin into ceramide and phosphorylcholine. *SMPD4* specifically encodes one of 4 neutral sphinogmyelinases, nSMase3 (MIM: [610457](http://omim.org/entry/610457)).

So far, only approximately 30 cases of this rare disorder have been reported across over 12 families. Clinical phenotype data of these cases is largely heterogeneous with severe neurological complications and early demise a key feature. This makes the identification, diagnosis and ongoing management of cases a challenge for patients, their families and medical practitioners.

@ravenscroft2021neurogenetic performed a study over 190 probands with a diagnosis of arthrogryposis multiplex congenita, distal arthrogryposis, fetal akinesia deformation sequence or multiple pterygium syndrome. This study identified a novel missense variation in SMPD4 which at the time of whole exome sequencing was not well described in the literature. The impacted family from Melbourne exhibited features involving arthrogryposis multiplex congenita, complex brain malformations, small for gestation age and hypoplasia of the corpus callosum. In two of the three related cases were additional features of microcephaly, congenital encephalopathy, cerebellar malformation and hypoplasia and hypomyelination.

@monies2019lessons described two families with different homozygous truncating variants in SMPD4 as a syndrome of skeletal dysplasia with cerebella atrophy. The phenotypes described involved bilateral clenched hands, talipes, IUGR, partial absence of corpus callosum with family history of three neonatal deaths with similar features.

A detailed study by @magini2019loss involved 12 unrelated familes with 32 individuals (21 with detailed clinical information). Hallmark presentations were of microcephaly, simplified gyration, hypomyelination, thin corpus callosum, mild cerebellar hypoplasia, brainstem hypoplasia, congenital arthrogryposis, diabetes mellitus, heart disease, severe encephalopathy and respiratory problems often leading to early demise. Despite this being the largest cohort studied, the clinical features and survival times among participants varied greatly. Three missense changes were noted in the study with affected children in these families often showing a milder presentation suggestive of possible residual function. In these cases individuals were able to develop independent motor skills, have mild intellectual disability and arthrogryposis without evidence of simplified gyral patterns on brain MRI. Other patients with truncating variants are shown to have more severe presentations, while a range of additional significant clinical features were reported involving dysmorphic facial features, seizure, vocal cord paralysis and hearing impairment.

A recent case study from China is described in @ji2022case involving a girl presenting in infancy with intrauterine growth restriction, microcephaly, postnatal developmental delay, arthrogryposis, hypertonicity, seizure, and hypomyelination on brain magnetic resonance imaging. @ji2022case argues the actual prevalence of this condition, based on the gene carrier rate is understated and may be attributable to typical symptoms of NEDMABA being non-specific, providing a diagnostic challenge in a clinical setting.

A recent study by @bijarnia2022growth presents the case of a 22-month old girl presenting with the typical phenotype of neurodevelopmental delay, prenatal onset growth failure, arthrogryposis, microcephaly and brain anomalies including severe hypomyelination, simplified gyral pattern and hypoplasia of corpus callosum and brainstem. Notably, there is also additional non-typical clinical findings of nystagmus and visual impairment secondary to macular dystrophy and retinal pigment epithelial stippling at posterior pole.

@ji2022case reports parallels with two cases showing the same homozygous null variant. An individual reported in @monies2019lessons presented with distinct symptoms of brain atrophy and skeletal dysplasia whereas the case in @magini2019loss with the same variant exhibited more typical clinical features.

Further work to collate and analyse data relating to NEDMABA is challenging. While @magini2019loss cataloged a detailed clinical phenotype data set as a supplementary data set to their study, these data are not suitable for statistical analysis as presented. Many of the clinical features are represented as free text descriptions and a variety of non-standard terminologies are applied between cases, making machine interpretation of the data difficult. Other studies [@ji2022case; @bijarnia2022growth] present only written case reports with some tabulated summaries. While larger studies [@ravenscroft2021neurogenetic; @monies2019lessons] focus more on genetic data, with less detail included on the clinical presentation of individuals in the cohort.

\
To overcome these challenges, @JSSv059i10 proposes a data structure know as 'tidy' data which organises each observation in a row, with each feature as its own column and each value in just one cell. In this case detailed text-based data can be tranformed to high dimensional binary indicators of key clinical features. This structure is conducive to effective statistical analysis and integrates deliberately with the *tidyverse* [@tidyverse] collection of packages for data analysis in R [@rbase].

Further analysis to better describe rare genetic conditions was explored in @diaz2020phenotype in the context of large scale genotype-phenotype analysis. The authors argue patients with rare disorders (often in small samples) often present with varied symptoms that do not match exactly with the described phenotype. When considering this situation of low sample size data with many features, methods in the field of multivariate statistical analysis are commonly deployed. Here the aim is to find substructure in the data, or a simple representation of the multi-dimensional space. Methods, such as Multiple Correspondence Analysis [@le2010multiple] are appropriate for high dimensional data comprised of binary indicators. This technique is cited in many studies in the application of dimension reduction and clustering of comorbidities and phenotypes from disease [@han2018cluster; @costa2013use].

When the focus is on survival or mortality, methods in statistical survival analysis are well suited and commonly used in the field of genomics [@chen2014survival]. These methods in particular provide a mechanism for dealing with right-censoring, a common feature of clinical studies where the clinical outcome of interest may not be known by the end of the study period. Within the context of examining a single condition, @Crowe810 investigated comorbidity phenotypes and mortality risk in ischaemic heart disease patients. Here latent class analysis was performed to identify a small number of clusters in the patients which could then be included in survival analysis techniques to better understand the substructre and outcome of patients of this condition.

An open research question exists around the clinical pathway and survival time of children exhibiting bi-allelic loss-of-function variants in SMPD4. With current research highlighting a diverse and heterogeneous phenotype, the relationship and correlations between these diverse features is not well understood. Furthermore, the survival time of affected children is not well understood despite early-demise featuring as a severe outcome. While a connection has been highlighted between some missense variants and a milder presentation with longer survival, this hypothesis has not been analysed further.

This research has three key aims. First, to collate and transform the variety of early case reports and studies on clinical phenotype data for this novel variant into an analysis-ready *tidy* data set. Secondly, to conduct analysis on the associations between both patients and their clinical features. Finally, to statistically quantify the expected survival time of children with NEDMABA based on the current case reports.

This will be the first in-depth analysis of early studies of this novel variant, which aims to produce statistical findings to better describe and understand this condition. This research is significant as it will assist clinicians understand the typical and non-typical presentations of this challenging new condition. In addition, genetic counselling of families with affected children will benefit from enhanced analysis on outcomes of existing reported cases.

## Methods

### SMPD4 Data Package

The data contained in @magini2019loss "Summary of SMPD4-Related Clinical Phenotype" is an excel spreadsheet tabulating each of the 21 individuals from the study with clinical details recorded. The file has 21 columns (one for each individual), and 64 rows (one for each phenotype or clinical remark).

The data were read into R [@rbase] without any changes, so as to preserve the reproducibility of the data transformation steps.

The data were transposed to ensure it could be presented as *tidy* formatted data [@JSSv059i10]. This requires:

1.  Each variable (clinical phenotype) forms a column.
2.  Each observation (individual) forms a row.
3.  Each type of observational unit forms a table (every cell has just one item)

In many cases, key clinical information was entered as free-text descriptions, which rendered any attempt of meaningful analysis impractical (@tbl-nontidy). In these cases, the text was tokenised by separating the list of clinical observations at each comma and forming a binary indicator column noting its presence '1' or absence '0' (@tbl-tidy).

|                     | Family 1- Individual 1                                                                                     | Family 1- Individual 4                                      |
|------------------|--------------------------|----------------------------|
| Facial dysmorphisms | short palpebral fissures, large ears, simple helices, smooth philtrum, thin lips, bilateral simian creases | short palpebral fissures, receding forehead, thin upper lip |

: Example of non-tidy free-text descriptions of features {#tbl-nontidy}

| id                     | short palpebral fissures | large ears | simple helices | receding forehead | ... |
|------------|------------|------------|------------|------------|------------|
| Family 1- Individual 1 | 1                        | 1          | 0              | ...               |     |
| Family 1- Individual 4 | 1                        | 0          | 0              | 1                 | ... |

: Example of tidy formatted data where individuals are transposed into rows and text into binary indicators {#tbl-tidy}

In the case where two variables were formed from phenotypes that are considered to be synonymous, these were merged into one indicator column to prevent duplication e.g. {bilateral_cleft_lips, bilateral_cleft_lip, cleft_lip_b\_l}.

Some data type conversion and categorical level standardisation was performed to ensure the data were in consistent and appropriate data types. For example, 'Gender' was not consistently coded, and 'Birth Weight' was encoded as a text string rather than a more useful numeric format. (@tbl-data-types).

| Gender       | Birth Weight          |
|--------------|-----------------------|
| male         | 2175 grams (- 2.5 SD) |
| female       | 2045 g (-3 SD)        |
| Female       | 2300 gram (-2 SD)     |
| Female fetus | n.a.                  |

: Example of inconsistent coding or sub-optimal data types {#tbl-data-types}

The final dataset consisted of 21 observations (one per individual) and 152 variables (one per clinical feature).

This format was preserved and other case studies identified in the literature [@ravenscroft2021neurogenetic; @monies2019lessons; @bijarnia2022growth; @ji2022case] were manually entered to conform to this template to allow the data to be combined for further analysis.

These data sets were packaged into an R Package called SMPD4 [@smpd4-data] in order to allow for reproducibility and sharing. This can be downloaded and installed from github at <https://github.com/deanmarchiori/SMPD4>.

### Multiple Correspondence Analysis

Multiple Correspondence Analysis (MCA) is an analogy to Principle Component Analysis (PCA) for categorical data [@le2010multiple].

We let a data set $\mathbf{X}$ be comprised of a set of individuals $I$ and a set of features $Q$ such that the $q_{th}$ feature has $K_q$ levels. The sum of all categories $K = \sum_{q=1}^{Q}K_q$ defines the dimensionality of $\mathbf{X}$ as an $I\times K$ matrix.

Taking $\delta_{ik} = 1$ if subject $i$ has feature $k$ and $\delta_{ik} = 0$ if the subject does not, we are left with the completely disjnuctive table $\mathbf{X} = I\times K$ of $\{0,1\}$. Letting the sum of all entries of $\mathbf{X}$ be $N$, we have $\mathbf{Z} = N^{-1}\mathbf{X}$.

We can introduce two diagonal matricies $\mathbf{D_r} = diag(\mathbf{r})$ and $\mathbf{D_c} = diag(\mathbf{c})$ where $\mathbf{r}$ and $\mathbf{c}$ are the vectors of row sums and column sums of $\mathbf{Z}$ respectively.

Computing MCA involves taking the Singular Value Decomposition of:

$$
\mathbf{M = D_r^{-\frac{1}{2}}(Z-rc^T)D_c^{-\frac{1}{2}}} = \mathbf{P}\Delta\mathbf{Q^T}
$$ where $\Delta$ are the singular values and $\Lambda = \Delta^2$ is the matrix of eigenvalues.

This results in the row and column factor scores respectively as:

$$
\mathbf{F} = \mathbf{D_r^{-\frac{1}{2}}P}\Delta 
$$

$$
\mathbf{G = D_c^{-\frac{1}{2}}Q}\Delta
$$

The combined clinical phenotype data was subsetted to include only those features that were encoded as a binary indicator variable.

Only those features that appeared in more than one case were included to minimise the influence of non-related features in this analysis.

This resulted in a dataset $X_{28 \times 61}$ indicator matrix of 61 clinical features across 28 individuals.

Multiple Correspondence Analysis was then computed using the *FactoMineR* R package [@factominer].

### Survival Analysis

To model survival probability for all subjects in the combined data, the data are subsetted to include *survival_time* which is either the number of days the individual survived for, or the age in days at last follow up, and *deceased* a numeric indicator which is equal to 1 if the subject is deceased and 0 otherwise.

The Kaplan-Meier estimator [@kaplan1958nonparametric] is used to estimate the survival function of the data.

The estimator is calcualted as:

$$
\hat{S}(t) = \prod_{t_i \leq t}\frac{n_i - d_i}{n_i} 
$$ where $t_i$ is some event time, $d_i$ represents the number of events (here deceased subjects) and $n_i$ indicating the indivudals known to have survived up to $t_i$

The baseline estimator of all individuals was calculated using the *survival* package in R [@survival-book].

Next the Kaplan-Meier estimator was calculated, stratified by the reported type of Single nucleotide polymorphisms (SNP) reported in the literature. A test was performed to detect differences in the survival curves using methods from @harrington1982class, again implemented in the *survival* package.

Finally, a Cox proportional hazards regression model [@andersen1982cox] was fit on survival time with the variant type as the sole covariate. This was to detect and measure the Hazard ratio of variant type on the survival time in order to test the hypothesis that misense variant types exhibit significantly greater survival compared to other variant types.

## Results

The function of this section is to summarize general trends in the data without comment, bias, or interpretation. The results of statistical tests applied to your data are reported in this section although conclusions about your original hypotheses are saved for the Discussion section.

### MCA

The first dimension ($\lambda_! = 0.19$) accounts for 19.29% of the variance in the data 
with the first 4 dimensions of the MCA analysis accouting for 54% of the variance.   


```{r}
### MCA  
tar_load(mca)

# eigenvalue summary 
mca$eig |> 
  kable(digits = 2, caption = "eigenvalues and percentage of variance explained by the MCA principal axes")
```

The contribution of the categories can be ranked within each of the first four
MCA dimensions.  

The first dimension is dominated by highly specific facial dysmophisms such as 
upward sweep of hair, priminent lower lip & bulbous nose. Also significant are cardiomyopathy and mild vermian hypoplasia. This principal dimension also notably 
is categorized by the absence of simplified gyral pattern and abnormal EEG 
with or without seizure.  

The second MCA dimension also has highly contributory facial features such as short philtrum, hairline related dysmorphisms and spastic tatraparesis. This dimension also has a notable absence of typical features such as microcephaly, arthrogryposis and hypomyelination. 

The third dimension is categorised by feeding and respiratory dysfunction with 
vocal cord palsy, tracheostomy, tube feeding and GERD. The fourth dimension is best described by small for gestational age and structural brain anomalies. See @fig-ctr


```{r ctr-plot, fig.height=10, fig.width=10, warning=FALSE}
#| label: fig-ctr
#| fig-cap: "Top 20 feature categories from MCA analysis for the first four MCA dimensions. The red dashed line indicates the mean contribution value. Each feature may be present or absent as indicated by 1 or 0 respectively"
mca$var$contrib |> 
  as_tibble(rownames = 'vars') |> 
  select(1:5) |> 
  pivot_longer(cols = -vars, names_to = "dim", values_to = "contribution") |> 
  mutate(col = ifelse(str_detect(vars, "_0"), "absent", "present")) |> 
  group_by(dim) |> 
  top_n(20, wt = contribution) |> 
  mutate(vars = fct_reorder(vars, contribution)) |> 
  ggplot(aes(contribution, reorder_within(vars, by = contribution, within = dim), fill = col)) +
  geom_col() +
  scale_y_reordered()+
  scale_fill_grey(start = 0.8, end = 0.2) +
  facet_wrap(~dim, scales = "free", ncol = 2) +
  geom_vline(xintercept = 100/122, lty = 2, col = 'red') +
  guides(fill=guide_legend(title="Condition")) +
  labs(y = '')
```

```{r fig.height=8, fig.width=8,  warning=FALSE}
fviz_mca_biplot(mca,  
               repel = T) 
```

### Cluster Analysis

```{r}
### Clustering 
tar_load(cluster_data)
tar_load(cluster_select)
tar_load(clustering)
tar_load(cluster_assignments)
```

```{r}
# CLValidation table or plot 
summary(cluster_select)
```

```{r}
cluster_select@measures |> 
  as_tibble(rownames = 'measure') |> 
  pivot_longer(cols =  -measure, names_to = 'cluster', values_to = "metric") |> 
  mutate(cluster = as.numeric(str_remove(cluster, ".hierarchical"))) |> 
  ggplot(aes(cluster, metric, colour = measure, group = measure)) +
  geom_line()
```

```{r, fig.height=12, fig.width=10}
# AGNES 
plot(clustering, which.plot = 2)
```

```{r, fig.height=8, fig.width=8,  warning=FALSE}
# plot MCA axes  
cbind(cluster_data, clust = cluster_assignments) |> 
  rownames_to_column(var = 'var') |> 
  as_tibble() |> 
  arrange(var, clust) |> 
  ggplot(aes(dim_1, dim_2, colour = clust)) +
  geom_point(position = "jitter") +
  geom_label_repel(aes(label = var), size = 2, max.overlaps = 20)

```

```{r}
# Profile table 
cbind(cluster_data, clust = cluster_assignments) |> 
  rownames_to_column(var = 'var') |> 
  as_tibble() |> 
  select(clust, var) |> 
  arrange(clust) |> 
  kable()
```

### Survival Analysis

```{r}
### Survival
tar_load(survival_data)
tar_load(kapm_baseline)
tar_load(kapm)
```

```{r}
# KM Curve
x <- survminer::surv_summary(kapm_baseline, data = survival_data)
kable(x, digits = 2)
```

```{r, fig.height=8, fig.width=8,  warning=FALSE}
survminer::ggsurvplot_df(x,
                         conf.int = TRUE,
                         risk.table = TRUE, # Add risk table
                         risk.table.col = "strata", # Change risk table color by groups
                         linetype = "strata", # Change line type by groups
                         surv.median.line = "hv", # Specify median survival
                         ggtheme = theme_bw())

```

```{r, fig.height=8, fig.width=8,  warning=FALSE}
# KM by variant

# box plot of variant 
ggplot(survival_data, aes(variant, survival_time)) +
  geom_boxplot()
```

```{r}
x2 <- survminer::surv_summary(kapm, data = survival_data)
kable(x2, digits = 2)
```

```{r, fig.height=8, fig.width=8,  warning=FALSE}
survminer::ggsurvplot_df(x2,
                              conf.int = TRUE,
          risk.table = TRUE, # Add risk table
          risk.table.col = "strata", # Change risk table color by groups
          linetype = "strata", # Change line type by groups
          surv.median.line = "hv", # Specify median survival
          ggtheme = theme_bw())

```

```{r}
# log rank
tar_load(logrank)
logrank

```

```{r}
# cox
tar_load(cox_ph)
cox_ph
```

## Discussion

```{r}
# table with med gen 
# cox power analysis
# cluster for outlier
# cluster for partitioning
```

-   the relationship between the results and the original hypothesis, i.e., whether they support the hypothesis, or cause it to be rejected or modified

-   an integration of your results with those of previous studies in order to arrive at explanations for the observed phenomena

-   possible explanations for unexpected results and observations, phrased as hypotheses that can be tested by realistic experimental procedures, which you should describe

-   End the Discussion with a summary of the principal points you want the reader to remember

\newpage

## References
