---
title: "Meta-Analysis of Clinical Phenotype and Patient Survival in Neurodevelopmental Disorder with Microcephaly, Arthrogryposis, and Structural Brain Anomalies Due to Bi-allelic Loss of Function Variants in SMPD"
author: Dean Marchiori
format:
  pdf:
    keep-tex: true
editor: visual
bibliography: bibliography.bib
date: last-modified
execute: 
  echo: false 
---

```{r echo=FALSE, message=FALSE}
library(targets)
source(here::here('_targets_packages.R'))
source(here::here('R', 'utils.R'))
theme_set(theme_bw())
```

# Abstract

A recently described, rare genetic condition known as Neurodevelopmental Disorder with Microcephaly, Arthrogryposis, and Structural Brain Anomalies (NEDMABA) has been identified in children with bi-allelic loss-of-function variants in *SMPD4*. The progression of this condition is not well understood with the limited case reports described so far exhibiting a severe and clinically diverse phenotype. A gap exists in the understanding of associations present in the heterogenous features of the clinical phenotype, and the expected survival probabilities of affected individuals. This is driven in part to the paucity of analysis-ready data on reported cases. This analysis aims to collate and standardise available case reports into a common dataset, to analyse and identify meaningful clusters in the clinical phenotype, and to quantify the survival probability for children with NEDMABA. To overcome the challenge of multidimensional data on very few subjects, we employ Multiple Correspondence Analysis (MCA) as a dimension reduction technique, which is then subject to cluster analysis and interpretation. To account for censoring in the data, Kaplan-Meier estimation is formulated to calculate patient survival time. The analysis correctly detected the classic phenotype for this condition, as well as a new distinct feature-cluster relating to findings of vocal cord paralysis, feeding dysfunction and respiratory failure. The survival probability for those affected was found to decline sharply in early infancy with median survival of 150 days, but with some surviving as long as 12.5 years. This wide range of outcomes is provisionally associated with different variant types however this conclusion could not be validated based on very low sample sizes. An `R` package called `SMPD4` was developed to publish standardised analysis-ready datasets used in this study. This analysis represents the first of its kind to help describe associations and trajectories of individuals with this newly reported condition, despite challenges with sparse and inconsistent data. This analysis can provide clinicians and genetic counsellors with better information to aide in decision making and support for families with this rare condition.

# Introduction

Neurodevelopment disorders are a diverse and heterogeneous group of conditions impacting the development of the nervous system, brain function, physical development and learning ability. A recently identified condition known as Neurodevelopmental Disorder with Microcephaly, Arthrogryposis, and Structural Brain Anomalies (NEDMABA) (MIM:[618622](https://omim.org/entry/618622)) has been described in children with bi-allelic loss-of-function variants in *SMPD4* [@magini2019loss]. Sphingomyelinases such as *SMPD4* play an important cellular role by hydrolyzing sphingomyelin into ceramide and phosphorylcholine. *SMPD4* specifically encodes one of 4 neutral sphinogmyelinases, nSMase3 (MIM: [610457](http://omim.org/entry/610457)). So far, less than 50 cases of this rare disorder have been reported in the literature. Clinical phenotype data of these cases is largely heterogeneous with severe neurological complications and early demise a key feature. This makes the identification, diagnosis and ongoing management of cases a challenge for patients, their families and medical practitioners.

A detailed study by @magini2019loss involved 12 unrelated families with 32 individuals (21 with detailed clinical information). Presentations were of microcephaly, simplified gyral pattern of the cortex, hypomyelination, cerebellar hypoplasia, congenital arthrogryposis, and early fetal/postnatal demise. Despite this being the largest cohort studied, the clinical features and survival times among participants varied greatly. Three missense changes were noted in the study with affected children in these families often showing a milder presentation suggestive of possible residual function. In these cases individuals were able to develop independent motor skills, have mild intellectual disability and arthrogryposis without evidence of simplified gyral patterns on brain MRI. Other patients with truncating variants are shown to have more severe presentations, while a range of additional significant clinical features were reported involving dysmorphic facial features, seizure, vocal cord paralysis and hearing impairment. @ravenscroft2021neurogenetic studied a family from Melbourne exhibiting features involving arthrogryposis multiplex congenita, complex brain malformations, small for gestation age and hypoplasia of the corpus callosum. In two of the three related cases were additional features of microcephaly, congenital encephalopathy, cerebellar malformation and hypoplasia and hypomyelination. A case study from China is described in @ji2022case involving a girl presenting in infancy with intrauterine growth restriction, microcephaly, postnatal developmental delay, arthrogryposis, hypertonicity, seizure, and hypomyelination on brain magnetic resonance imaging. The authors report parallels with two cases showing the same homozygous null variant. An individual reported in @monies2019lessons presented with distinct symptoms of brain atrophy and skeletal dysplasia whereas the case in @magini2019loss with the same variant exhibited more typical clinical features. A recent study by @bijarnia2022growth presents the case of a 22-month old girl presenting with the typical phenotype of neurodevelopmental delay, prenatal onset growth failure, arthrogryposis, microcephaly and brain anomalies including severe hypomyelination, simplified gyral pattern and hypoplasia of corpus callosum and brainstem. Notably, there was also additional non-typical clinical findings of nystagmus and visual impairment secondary to macular dystrophy and retinal pigment epithelial stippling at posterior pole.

Further work to collate and analyse data relating to NEDMABA is challenging. While @magini2019loss included a supplementary clinical phenotype data set in their study, these data are not suitable for statistical analysis. Many of the clinical features are represented as text descriptions and a variety of non-standard terminologies are applied between cases, making machine interpretation of the data difficult. Other studies [@ji2022case; @bijarnia2022growth] present only written case reports with some tabulated summaries. While larger studies [@ravenscroft2021neurogenetic; @monies2019lessons] focus more on genetic data, with less detail included on the clinical presentation of individuals. To overcome these data challenges, @JSSv059i10 proposes a data structure know as 'tidy' data which organises each observation in a row, with each feature as its own column and each value in just one cell. In this case detailed text-based data can be transformed to high dimensional binary indicators of key clinical features. This structure is conducive to effective statistical analysis and integrates deliberately with the *tidyverse* [@tidyverse] collection of packages for data analysis in R [@rbase].

Further analysis to better describe rare genetic conditions was explored in @diaz2020phenotype in the context of large scale genotype-phenotype analysis. The authors argue patients with rare disorders (often in small samples) frequently present with varied symptoms that do not match exactly with the described phenotype. When considering the situation of low sample size data with many features, methods in the field of multivariate statistical analysis are commonly deployed. Here the aim is to find substructure in the data, or a simple representation of the multi-dimensional space. Methods, such as Multiple Correspondence Analysis [@le2010multiple] are appropriate for high dimensional data comprised of binary indicators. This technique is cited in many studies in the application of dimension reduction and clustering of comorbidities and phenotypes from disease [@han2018cluster; @costa2013use].

In terms of patient survival, methods in statistical survival analysis are well suited and commonly used in the field of genomics [@chen2014survival]. These methods in particular provide a mechanism for dealing with right-censoring, a common feature of clinical studies where the clinical outcome of interest may not be known by the end of the study period.

An open research question exists around the clinical pathway and survival time of children exhibiting bi-allelic loss-of-function variants in *SMPD4*. With current research highlighting a diverse and heterogeneous phenotype, the relationship between these diverse features is not well understood. Furthermore, the survival time of affected children has not been extensively analysed despite early-demise featuring as a severe outcome. While a connection has been highlighted between some missense variants and a milder presentation with longer survival [@magini2019loss], this hypothesis has not been analysed further.

This analysis has three key aims. First, to collate and transform the variety of early case reports and studies on clinical phenotype data for this novel variant into an analysis-ready *tidy* data set. Secondly, to conduct analysis on the associations between commonly reported clinical features. Finally, to statistically quantify the expected survival time of children with NEDMABA based on the current case reports.

This will be the first in-depth analysis of this novel variant, which aims to produce statistical findings to better understand this newly described condition. This research will assist clinicians understand the various presentations of this challenging new condition. In addition, genetic counselling of families with affected children will benefit from enhanced analysis on outcomes of existing reported cases.

# Methods

## SMPD4 Data Package

The data contained in @magini2019loss "Summary of SMPD4-Related Clinical Phenotype" is an excel spreadsheet tabulating each of the 21 individuals from the study with clinical details recorded. The file has 21 columns (one for each individual), and 64 rows (one for each phenotype or clinical remark).

The data were read into `R` [@rbase] without any changes, so as to preserve the reproducibility of the data transformation steps. The data were transposed to ensure it could be presented as *tidy* formatted data [@JSSv059i10]. This requires that each variable (clinical phenotype) forms a column; each observation (individual) forms a row and each type of observational unit forms a table (every cell has just one item). This resulted in a long-formatted data set of 21 rows (individuals) and 64 columns (clinical features). In many cases, key clinical information was entered as free-text descriptions, which rendered any attempt of meaningful analysis impractical (@tbl-nontidy). In these cases, the text was tokenised by separating the list of clinical observations at each comma and forming a binary indicator column noting its presence '1' or absence '0' (@tbl-tidy).

```{r}
#| label: tbl-nontidy
#| tbl-cap: Example of non-tidy free-text descriptions of features.
tibble(
  `Family 1- Individual 1` = c("short palpebral fissures, large ears, simple helices"),
  `Family 1- Individual 4 ` = c("short palpebral fissures, receding forehead")
  ) |> 
  gt()
```

```{r}
#| label: tbl-tidy
#| tbl-cap: Example of tidy formatted data where individuals are transposed into rows and text into binary indicators
tribble(
  ~"id", ~"short palpebral fissures", ~"large ears", ~"simple helices", ~"receding forehead", 
  "Family 1- Individual 1", 1, 1, 1, 0,
  "Family 1- Individual 4", 1, 0, 0, 1, 
  ) |> 
  gt()
```

In the case where synonymous terms are present, these indicator columns are consolidated and merged into one indicator column to prevent duplication e.g. {bilateral_cleft_lips, bilateral_cleft_lip, cleft_lip_b\_l} -\> {bilateral_cleft_lip}.

Data type conversion and categorical level standardisation was performed to ensure the data were in consistent and appropriate data types. For example, 'Gender' was not consistently coded, and 'Birth Weight' was encoded as a text string rather than a more useful numeric format. (@tbl-data-types).

| Gender       | Birth Weight          |
|--------------|-----------------------|
| male         | 2175 grams (- 2.5 SD) |
| female       | 2045 g (-3 SD)        |
| Female       | 2300 gram (-2 SD)     |
| Female fetus | n.a.                  |

: Example of inconsistent coding or sub-optimal data types {#tbl-data-types}

| Gender | Birth Weight (g) |
|--------|------------------|
| male   | 2175             |
| female | 2045             |
| female | 2300             |
| female | NA               |

: Example of consistent and appropriate coding or data types {#tbl-data-types2}

This format was used as a template to align other written case reports identified in the literature [@ravenscroft2021neurogenetic; @monies2019lessons; @bijarnia2022growth; @ji2022case]. These were manually entered to conform to this template to allow the data to be combined for further analysis.

The final data set consisted of 28 observations (one per individual) and 152 variables (one per clinical feature). These variables were comprised of qualitative text descriptions, binary indicators of features as described above and other fields including background information on the patient and various bio markers. A full list of this data set is available in @tbl-dict. These data sets were compiled into a publicly available `R` Package called `SMPD4` [@smpd4-data] to allow for reproducibility and sharing.

## Multiple Correspondence Analysis

The data on subjects above is subset to include only variables that can be transformed into binary indicator variables representing the presence or absence of a given clinical feature. Only those features that appeared in more than one case report were included to minimise the influence of isolated features.

Dimension reduction on this wide data set was performed using Multiple Correspondence Analysis (MCA) (an analogy to Principle Component Analysis (PCA) for categorical data) [@le2010multiple] in order to arrive at a set of meaningfully small dimensions that account for most of the variation in the data. This computation was carried out using the *FactoMineR* R package [@factominer].

We let these data be $\mathbf{X}$ which is comprised of a set of individuals $I$ and a set of features $Q$ such that the $q_{th}$ feature has $K_q$ levels. The sum of all categories $K = \sum_{q=1}^{Q}K_q$ defines the dimensionality of $\mathbf{X}$ as an $I\times K$ matrix. This resulted in a data set $X_{28 \times 61}$ indicator matrix of 61 clinical features across 28 individuals. Taking $\delta_{ik} = 1$ if subject $i$ has feature $k$ and $\delta_{ik} = 0$ if the subject does not, we are left with the completely disjunctive table $\mathbf{X} = I\times K$ of $\{0,1\}$. Letting the sum of all entries of $\mathbf{X}$ be $N$, we have $\mathbf{Z} = N^{-1}\mathbf{X}$. We can introduce two diagonal matrices $\mathbf{D_r} = diag(\mathbf{r})$ and $\mathbf{D_c} = diag(\mathbf{c})$ where $\mathbf{r}$ and $\mathbf{c}$ are the vectors of row sums and column sums of $\mathbf{Z}$ respectively.

Computing MCA involves taking the Singular Value Decomposition:

$$
\mathbf{M = D_r^{-\frac{1}{2}}(Z-rc^T)D_c^{-\frac{1}{2}}} = \mathbf{P}\Delta\mathbf{Q^T}
$$ where $\Delta$ are the singular values and $\Lambda = \Delta^2$ is the matrix of eigenvalues.

This results in the row and column factor scores respectively as:

$$
\begin{split}
\mathbf{F} = \mathbf{D_r^{-\frac{1}{2}}P}\Delta \\
\mathbf{G = D_c^{-\frac{1}{2}}Q}\Delta
\end{split}
$$

The *contribution* of each category was computed for each of the principal axis per @le2010multiple. This metric identifies the proportion of variance of that axis due to the point and is defined as:

$$
Ctr_k = \frac{py_k^2}{\lambda}
$$ Where $p$ represents the point weighting, $y$ is the coordinate relative to the principal axes of variance $\lambda$.

The coordinates on the first few principal axes were inspected for any outliers or unusual data before being graphically interpreted and subject to further analysis.

## Cluster Analysis

The MCA coordinates for the first 4 principal dimensions were subject to clustering in order to find groups of features that were internally homogeneous and externally heterogeneous. This was computed in `R` using the `clValid` package [@clvalid].

A range of clustering algorithms were compared including AGNES hierarchical agglomerative [@kaufman2009finding], k-means [@hartigan1979algorithm] and self-organising maps (SOM) [@kohonen2012self]. A sensible range of clusters from 2-5 were trialed for each clustering algorithm. The results for each method were evaluated using three measures of stability: Average proportion of non-overlap (APN), Average Distance (AD) and Average Distance Between Means (ADM) [@datta2003comparisons]. Stability measures compare the full clustering result with a result based on dropping each of the columns one at a time. These metrics were selected to represent each data point (clinical feature) with a number of MCA dimensions. A good cluster solution is one that is robust and meaningful across all of these dimensions ([@fig-clvalid]). The optimal method and cluster number was selected based on a trade off between the best performing method using the evaluation metrics and a sensible partitioning of the data for this analysis ([@tbl-eig]).

## Survival Analysis

To model survival probability for all subjects in the combined data, the data were subsetted to include `survival_time` which is either the number of days the individual survived for, or the age in days at last follow up, and `deceased` a numeric event indicator which is equal to 1 if the subject is deceased and 0 if the subject was still alive at the time the respective study had concluded. The Kaplan-Meier estimator [@kaplan1958nonparametric] was used to estimate the survival function of the data.

The estimator is calculated as:

$$
\hat{S}(t) = \prod_{t_i \leq t}\frac{n_i - d_i}{n_i} 
$$ where $t_i$ is some event time, $d_i$ represents the number of events (here deceased subjects) and $n_i$ indicating the individuals known to have survived up to $t_i$

The baseline estimator of all individuals was calculated using the `survival` package in `R` [@survival-book]. Next the Kaplan-Meier estimator was stratified by the type of genetic variation reported in the literature. A log-rank test was performed to detect differences in the survival curves using methods from @harrington1982class, again implemented in the `survival` `R` package.

# Results

## MCA

The initial results of the MCA were inspected with a bi-plot and identified two subjects (17, 18) from the data that were clear outliers and sat significantly beyond the 95% confidence ellipse (@fig-biplotprelim). These individuals represented two twins from @magini2019loss who were described with a much milder phenotype and were subject to a missense variation in *SMPD4*. Further analysis of this influence is conducted below. For now, these subjects were removed to allow for more meaningful analysis of other features.

Repeating MCA for the remaining subjects shows the first dimension ($\lambda_1 = 0.18$) accounts for 17.88% of the variance in the data with the first 4 dimensions of the MCA analysis accounting for 54% of the variance (@tbl-eig).

```{r}
tar_load(mca_prelim) 
```

```{r  warning=FALSE}
#| label: fig-biplotprelim
#| fig-cap: "A bi-plot of the first two principal axes with grey circles indicating the individual's projected position and black triangles indicating the variables. A 95% confidence ellipse is plotted in red around the origin."
fviz_mca_biplot(
  mca_prelim,
  axes = c(1, 2),
  geom = c("point", "text"),
  geom.ind = c("point", "text"),
  geom.var = c("point", "text"),
  repel = TRUE,
  label = "all",
  invisible = "none",
  habillage = "none",
  addEllipses = T,
  ellipse.level=0.95,
  select.var = list(name = NULL, cos2 = 0.2, contrib = NULL),
  palette = "grey",
  col.ind = "red",
  col.var = "black",
  arrows = c(FALSE, FALSE),
  map = "symmetric",
  title = ""
)
```

```{r}
#| label: tbl-eig
#| tbl-cap: Eigenvalues and percentage of variance explained by the MCA principal axes

### MCA  
tar_load(mca)

mca$eig |> 
  tibble::as_tibble(rownames = 'dimension') |> 
  gt() |> 
  fmt_number(columns = 2,
             decimals = 2) |> 
  fmt_percent(columns = c(`percentage of variance`, `cumulative percentage of variance`),
              scale_values = F)
```

The *contribution* of the categories were ranked within each of the first four principal dimensions in @fig-ctr. The first dimension is dominated by isolated cases with highly specific facial dysmorphisms such as epicanthal folds, short philtrum and low anterior hairline. The second MCA dimension is categorised by feeding and respiratory dysfunction with vocal cord palsy, tracheostomy, tube feeding and GERD. Examining the bi-plot of the first two principal axes (@fig-biplot) illustrates the contrasting sources of variation between conditions relating to airway and swallowing function and predominantly facial features and structural brain anomalies such as as cerebellar hypoplasia, development delay. The third dimension is characterised by atypical, complex features such as patent ductus arteriosus, respiratory infections, small for gestational age and further facial features. These are contrasted by the fourth principal dimension which is strongly categorised by respiratory distress including stridor and hypercapnia in the absence of typical presentations of simplified gyral pattern, intrauterine growth retardation and microcephaly.

```{r ctr-plot, warning=FALSE}
#| label: fig-ctr
#| fig-cap: "Top 5 feature categories as measured by feature contribution from MCA analysis for the first four MCA dimensions. The red dashed line indicates the mean contribution value."
mca$var$contrib |> 
  as_tibble(rownames = 'vars') |> 
  select(1:5) |> 
  pivot_longer(cols = -vars, names_to = "dim", values_to = "contribution") |> 
  mutate(col = ifelse(str_detect(vars, "_0"), "absent", "present")) |> 
  group_by(dim) |> 
  slice_max(n = 5, order_by = contribution, with_ties = F) |> 
  mutate(vars = fct_reorder(vars, contribution)) |> 
  mutate(vars = str_remove(vars, "_[:digit:]")) |> 
  mutate(vars = str_remove(vars, "_ind")) |> 
  mutate(vars = str_replace_all(vars, "_", " ")) |> 
  mutate(vars = str_wrap(vars, width = 25)) |> 
  ggplot(aes(contribution, reorder_within(vars, by = contribution, within = dim))) +
  geom_col() +
  scale_y_reordered() +
  scale_fill_grey(start = 0.8, end = 0.2) +
  facet_wrap(~dim, scales = "free", ncol = 2) +
  geom_vline(xintercept = 100/122, lty = 2, col = 'red') +
  theme(legend.position = 'bottom', axis.text = element_text(size = 8)) +
  labs(y = '', x = '')
```

```{r , warning=FALSE}
#| label: fig-biplot
#| fig-cap: "A bi-plot of the first two principal axes with grey circles indicating the individual's projected position and black triangles indicating the variables. A 95% confidence ellipse is plotted in grey around the origin."
fviz_mca_biplot(
  mca,
  axes = c(1, 2),
  geom = c("point", "text"),
  geom.ind = c("point"),
  geom.var = c("point", "text"),
  repel = TRUE,
  label = "all",
  invisible = "none",
  habillage = "none",
  addEllipses = T,
  ellipse.level=0.95,
  select.var = list(name = NULL, cos2 = 0.4, contrib = NULL),
  palette = "grey",
  col.ind = "grey80",
  col.var = "black",
  arrows = c(FALSE, FALSE),
  map = "symmetric",
  title = ""
)
```

```{r ,  warning=FALSE}
#| label: fig-biplot2
#| fig-cap: "A bi-plot of principal axes 3 and 4, with grey circles indicating the individual's projected position and black triangles indicating the variables. A 95% confidence ellipse is plotted in grey around the origin."
fviz_mca_biplot(
  mca,
  axes = c(3, 4),
  geom = c("point", "text"),
  geom.ind = "point",
  geom.var = c("point", "text"),
  repel = TRUE,
  label = "all",
  invisible = "none",
  habillage = "none",
  addEllipses = T,
  ellipse.level=0.95,
  select.var = list(name = NULL, cos2 = 0.4, contrib = NULL),
  palette = "grey",
  col.ind = "grey80",
  col.var = "black",
  arrows = c(FALSE, FALSE),
  map = "symmetric",
  title = ""
)

```

## Cluster Analysis

```{r}
### Clustering 
tar_load(cluster_data)
tar_load(cluster_select)
tar_load(clustering)
tar_load(cluster_assignments)
```

Cluster validation measures resulted in either 2 or 5 clusters as the optimal choice (@tbl-opticlust). In order to partition the data into meaningful groupings, k-means with 5 clusters was selected. This reflects the optimal choice for both AD and FOM cluster metrics. The results are summarised in @tbl-clusttab and a scatter plot of the cluster results is shown in @fig-clust is projected on the first two principal dimensions. This highlights a core cluster of classic features of NEDMABA such as microcephaly, hypomyelination, arthrogryposis and feet deformity and structural brain anomalies. A separate and adjacent cluster is formed of airway and feeding related conditions involving vocal cord paralysis, tracheostomy, tube feeding and enlarged ventricles. Three other smaller satellite clusters exists where highly distinctive features were associated with a single individual or family. A full articulation of feature to cluster assignment is given in @tbl-profile.

```{r}
#| label: tbl-opticlust
#| tbl-cap: "Optimal cluster numbers and methods based on a range of stability validation metrics."
optimalScores(cluster_select) |> 
  as_tibble(rownames = 'Metric') |> 
  gt() |> 
  fmt_number(columns = Score, decimals = 2)
```

```{r, warning=FALSE}
#| label: fig-clvalid
#| fig-cap: "cluster validation analysis showing various metrics for cluster evaluation from 2 to 5 clusters. The most optimal number of clusters are selected by picking the lowest value from one or many of these comparable metrics."

cluster_select@measures |>
  as_tibble(rownames = 'measure') |>
  pivot_longer(cols =  -measure, names_to = 'cluster', values_to = "metric") |>
  separate(col = cluster, into = c('k', 'method'), sep = "\\.") |> 
  mutate(k = as.numeric(k)) |>
  ggplot(aes(k, metric, colour = method)) +
  geom_line() +
  labs(x = 'number of clusters') +
  facet_wrap(~measure, scales= 'free')
```

```{r}
#| label: tbl-clusttab
#| tbl-cap: "A summary table to cluster membership showing the cluster number and number of features in each cluster and a sample of features that have membership in each."
cluster_assignments |> 
  as_tibble(rownames = 'feature') |> 
  mutate(feature = str_remove(feature, "_ind_1|_1"),
         feature = str_replace_all(feature, "_", " ")) |> 
  mutate('Cluster No.' = value) |> 
    mutate("Cluster Description" = fct_recode(as.character(value),
    Core = "4",
    Adjacent = "2",
    Outlying = "3",
    Outlying = "5",
    Outlying = "1",
    )) |> 
  group_by(`Cluster No.`, `Cluster Description`) |> 
  summarise('No. Features' = n(),
            `Sample Features in Cluster` = paste(head(feature, 4), collapse = ", "), .groups = 'drop') |> 
  select(-`Cluster Description`) |> 
  gt() |> 
  gt::tab_footnote(footnote = "A sample of 4 features shown to illustrate cluster membership", locations = cells_column_labels("Sample Features in Cluster")) 
```

```{r,warning=FALSE}
#| label: fig-clust
#| fig-cap: "A projection of features on the first two principal components with k-means cluster assignment indicated by colour. A random sample of three labels per cluster are printed to provide a representation of each cluster's characteristics."

d <- cbind(cluster_data, clust = factor(cluster_assignments)) |> 
  rownames_to_column(var = 'var') |> 
  as_tibble() |> 
  arrange(var, clust) |> 
  mutate(clust = fct_recode(clust,
    Core = "4",
    Adjacent = "2",
    Outlying_3 = "3",
    Outlying_5 = "5",
    Outlying_1 = "1",
    ))

hull <- d %>%
  group_by(clust) |> 
  slice(chull(dim_1, dim_2))

  ggplot(d, aes(dim_1, dim_2, colour = clust)) +
  geom_polygon(data = hull, alpha = 0.2, aes(fill = clust)) +
  geom_vline(xintercept = 0, lty = 2, colour = "grey50") +
  geom_hline(yintercept = 0, lty = 2, colour = "grey50") +
  geom_label_repel(data = d |> group_by(clust) |> slice_sample(n=3), aes(label = var), size = 2, show.legend = F) +
  geom_point(aes(colour = clust), position = "jitter", show.legend = F) +
  labs(x = "Dimension 1",
       y = "Dimension 2",
       fill = 'cluster') +
    guides(color='none') +
    theme(legend.position = 'none')
```

## Survival Analysis

```{r}
### Survival
tar_load(survival_data)
tar_load(kapm_baseline)
tar_load(kapm)

x <- survminer::surv_summary(kapm_baseline, data = survival_data)
```

The Kaplan-Meier estimator shows a range of survival times (0, 4562 days) with median survival time calculated as 150 days. At time 0 the survival probability was 0.82 (95% CI: 0.69, 0.98). Survival probability after one year of 0.403 (95% CI: 0.252, 0.644) and survival probability at close to ten years of 0.24 (95% CI: 0.11, 0.52) See @tbl-km.

```{r, warning=FALSE}
#| label: fig-surv
#| fig-cap: "Kaplan Meier curve with the black line representing the survival probability and with grey shading indicating the 95% confidence interval. Cross icons represent right-censoring"
survminer::ggsurvplot_df(x,
                         color = "black",
                         conf.int = TRUE,
                         conf.int.fill = "gray",
                         risk.table = TRUE, # Add risk table
                         risk.table.col = "strata", # Change risk table color by groups
                         linetype = "strata", # Change line type by groups
                         surv.median.line = "hv", # Specify median survival
                         ggtheme = theme_bw(),
                         legend = "none") +
  labs(x = "time (days)")

```

A comparative box plot of survival times (@fig-box) by variant type shows significant outliers for compound heterozygous *missense + splice* and homozygous *missense* variants. However, even within homozygous missense variants the survival time varies greatly. Other missense variants, such as heterozygous missense with gene deletion do not show the same increased survival time as noted by @magini2019loss.

```{r, warning=FALSE}
#| label: fig-box
#| fig-cap: "A compariative box plot showing the range of survival times for subjects split by the type of variant. Green bars indicate homozygous variants, while red indicates compound heterozygous variants."

# box plot of variant 
ggplot(survival_data, aes(survival_time, variant,fill = variant_type.x)) +
  geom_boxplot() +
  theme(legend.position = 'bottom') +
  scale_fill_grey(start = 0.4, end = 0.9) +
  labs(fill = 'Variant Type',
       x = 'Survival Time',
       y = '')
```

A log-rank test was carried out to compare the survival curves of patients stratified by the various gene variants and showed a significant difference in the stratified survival curves ($\chi^2 = 14.8, p = 0.04)$ however due to the small sample size and large number of possible variants, several had expected counts less than 5 which would create a violation in assumptions for the test.

```{r}
# log rank
tar_load(logrank)
```

# Discussion

This meta-analysis of current research was able to consolidate data that allowed statistical analysis to be conducted on affected individuals. The use of dimension reduction techniques was shown to be successful in uncovering relevant and novel substructure in the data. This was particularly evident with the detection of the accepted typical phenotype of microcephaly, hypomyelination, arthrogryposis, simplified gyral pattern and developmental delay. After excluding two outlying remarkable cases, we identified the presence of an adjacent cluster of related features that are not well described in The Human Phenotype Ontology [@kohler2021human] as shown in @tbl-hpo. It shows the prolific feature of 'respiratory distress' as more commonly associated with these adjacent features such as vocal cord palsy, tracheostomy, tube feeding and Gastroesophageal reflux disease (GERD). These features are present in the case reported data, but not well described in HPO and help account for the variation seen in the clinical phenotype in the first few principal dimensions.

The heterogeneous variation in the data was clearly evident with the first four principal axes being required to explain more than 50% of the variation in the data set. Visual inspection shows the primary sources of variation in the clinical phenotype are explained by a few individuals with very different clinical features. Many of these exhibit highly specific facial or anatomical dysmorphisms. While coarse facial features are a core presentation, those with a milder course had clinical reports that articulated these dysmorphisms in much more detail. Additionally, talipes appears as an isolated feature without the typical microcephaly and hypomyelination, however there was significant heterogeneity in how feet contractures were reported in clinical case reports. This is particularly evident in rocker-bottom feet which was found closely related in our analysis to core features despite the prevalence in HPO being quite low (2/21), suggesting possible under-reporting or confounded reporting of this feature. Another core identified feature is bilateral cleft lip. This condition has a reported frequency of just 2/21 (@tbl-hpo) and may be an emerging feature for a more typical presentation given its close relationship with other classic features. Features isolated to just one or few cases in our consolidated data have been labelled as 'Isolated' in @tbl-hpo. An optimal choice of both 2 and 5 clusters were recommended in the analysis. A partition of the data with just two clusters would likely be useful in separating outliers from other data. In this analysis a larger number (5) was selected to align the goals of the analysis which was to uncover distinct groups of features.

In terms of patient survival, the first few months of life show a sharp decline in survival probability confirming early demise in the infant period as a key factor. A small number of cases show survival into childhood and in some cases into the second decade of life. This work identifies a one year survival rate of 0.403 (95% CI: 0.252, 0.644). On the other hand, a naive estimate using just the number of deceased patients at this time as proportion of the study population is 0.43 (1 - 16/28). This highlights the importance of applying a survival analysis technique that incorporates censoring so as not to erroneously overestimate the survival probability. The current literature suggests the type of gene variant has an impact on survival time, which seems to be supported in this work. Through a comparison of multiple survival curves, stratified by variation type, the limited sample size prevented definitive conclusions with many cells having expected counts less than 5.

Despite NEDMABA exhibiting a diverse and heterogeneous clinical phenotype, our analysis was able to identify meaningful substructures in this data through the use of dimension reduction techniques (MCA) and cluster analysis. This has isolated the classic NEDMABA phenotype of microcephaly, hypomyelination, arthrogryposis, simplified gyral pattern and developmental delay. Furthermore it has isolated additional distinct groupings of features that are exhibited in patients such as vocal cord paralysis, swallowing dysfunction, tube feeding and GERD. Furthermore, we were able to quantify the expected baseline survival probabilities for children with this condition as well as quantifying the significant variation in these estimates. Future work is required with the benefit of more cases, to better understand the differences in survival curves for other factors associated with genotype.

\newpage

# Appendix

```{r}
#| label: tbl-hpo
#| tbl-cap: "A mapping table of phenotype terms from The Human Phenotype Ontology (HPO) with clusters defined in this analysis. Not all clinical features defined in the analysis are present in the HPO terms"
#| 
hpo <- tibble::tribble(
                           ~`HPO Term Name`,               ~Category,   ~Cluster, ~Frequency,
                       "Adducted thumb",                 "Limbs", "Isolated",     "1/21",
             "Simplified gyral pattern",        "Nervous System",     "Core",    "12/18",
                           "Hypertonia",           "Musculature",     "Core",     "9/15",
                              "Seizure",        "Nervous System",     "Core",    "10/17",
                            "Hypotonia",           "Musculature",     "Core",     "4/15",
           "Global developmental delay",        "Nervous System",     "Core",      "7/7",
                "Highly arched eyebrow", "Skin, Hair, and Nails", "Isolated",     "2/21",
                "Cerebellar hypoplasia",        "Nervous System",     "Core",    "10/21",
        "Neonatal respiratory distress",    "Respiratory System", "Adjacent",    "14/17",
                          "Bifid uvula",         "Head and neck", "Isolated",     "1/21",
                        "Deep philtrum",         "Head and neck", "Isolated",     "1/21",
                  "CNS hypomyelination",        "Nervous System",     "Core",     "7/19",
                "Lumbar hypertrichosis", "Skin, Hair, and Nails", "Isolated",     "1/21",
  "Birth length less than 3rd percentile",                "Growth",     "Core",     "6/11",
          "Hypoplasia of the brainstem",        "Nervous System", "Isolated",     "3/20",
           "Tented upper lip vermilion",         "Head and neck", "Isolated",     "3/21",
                         "Hypotelorism",                   "Eye", "Isolated",     "1/21",
              "Short palpebral fissure",         "Head and neck", "Outlying",     "4/21",
                  "Bilateral cleft lip",         "Head and neck",     "Core",     "2/21",
                    "Diabetes mellitus",             "Endocrine", "Isolated",     "2/19",
      "Single transverse palmar crease", "Skin, Hair, and Nails", "Isolated",     "1/21",
                           "Epicanthus",         "Head and neck", "Outlying",     "5/21",
                "Low anterior hairline",         "Head and neck", "Outlying",     "2/21",
   "Arthrogryposis multiplex congenita",     "Connective tissue",     "Core",    "17/20",
             "Progressive microcephaly",        "Nervous System",     "Core",     "9/10",
                         "Microcephaly",        "Nervous System",     "Core",    "15/21",
             "Thin upper lip vermilion",         "Head and neck", "Outlying",     "2/21",
                "Thin vermilion border",         "Head and neck", "Isolated",     "3/21",
      "Intrauterine growth retardation",                "Growth",     "Core",    "12/19",
             "Posteriorly rotated ears",                   "Ear", "Adjacent",     "2/21",
                      "Narrow forehead",         "Head and neck", "Isolated",     "1/21",
                     "Sloping forehead",         "Head and neck", "Isolated",     "4/21",
                      "Smooth philtrum",         "Head and neck", "Isolated",     "2/21",
                       "Short philtrum",         "Head and neck", "Outlying",     "2/21",
                      "Premature birth",    "Prenatal and Birth",     "Core",     "7/18",
                "Mandibular prognathia",         "Head and neck", "Outlying",     "2/21",
         "Wide intermamillary distance",                "Breast", "Isolated",     "1/21",
               "Depressed nasal bridge",         "Head and neck", "Outlying",     "1/21",
       "Downslanted palpebral fissures",         "Head and neck", "Outlying",     "2/21",
                           "Short neck",         "Head and neck", "Isolated",     "1/21",
                         "Bulbous nose",         "Head and neck", "Isolated",     "2/21",
                       "Protruding ear",                   "Ear", "Outlying",     "2/21",
                   "Rocker bottom foot",                 "Limbs",     "Core",     "2/21"
  )

hpo |> 
  gt() |> 
   tab_source_note(
    source_note = "Source: The Human Phenotype Ontology (HPO) https://hpo.jax.org/app/browse/disease/OMIM:618622 "
  ) |> 
  cols_move_to_start(
    columns = c(`HPO Term Name`, Category, Frequency)
  ) |> 
   cols_label(
     Cluster = md("**Cluster**")
  ) |> 
   tab_spanner(
    label = "HPO Data",
    columns = c(`HPO Term Name`, Category, Frequency)
  ) |> 
  tab_spanner(
    label = "Derived Clusters",
    columns = c(Cluster)
  ) |> 
  tab_source_note(
    source_note = md("Reference: Köhler S, Gargano M, Matentzoglu N, Carmody LC, Lewis-Smith D, Vasilevsky NA, Danis D, Balagura G, Baynam G, Brower AM, Callahan TJ, Chute CG, Est JL, Galer PD, Ganesan S, Griese M, Haimel M, Pazmandi J, Hanauer M, Harris NL, Hartnett MJ, Hastreiter M, Hauck F, He Y, Jeske T, Kearney H, Kindle G, Klein C, Knoflach K, Krause R, Lagorce D, McMurry JA, Miller JA, Munoz-Torres MC, Peters RL, Rapp CK, Rath AM, Rind SA, Rosenberg AZ, Segal MM, Seidel MG, Smedley D, Talmy T, Thomas Y, Wiafe SA, Xian J, Yüksel Z, Helbig I, Mungall CJ, Haendel MA, Robinson PN. *The Human Phenotype Ontology in 2021.* Nucleic Acids Res. 2021 Jan 8;49(D1):D1207-D1217. doi: [10.1093/nar/gkaa1043](10.1093/nar/gkaa1043). PMID: 33264411; PMCID: PMC7778952.")
  ) |> 
  tab_footnote(
    footnote = "Clusters defined in this analysis mapped to HPO Terms",
    locations = cells_column_labels(columns = "Cluster")
  )
```

```{r}
#| label: tbl-profile
#| tbl-cap: Assignment of clinical features with their cluster. These clusters represent similar features projected onto the first four MCA principal axes. Clusters that are on opposing sides of an axes contract with each other and account for the largest amount of variation in the data. 
cbind(cluster_data, cluster = cluster_assignments) |> 
  rownames_to_column(var = 'variable') |> 
  as_tibble() |> 
  select(cluster, variable) |> 
  mutate(variable = str_remove(variable, "_[:digit:]")) |> 
  mutate(variable = str_remove(variable, "_ind")) |> 
  arrange(cluster) |> 
  kable()
```

```{r}
#| label: tbl-km
#| tbl-cap: "Survival probabilities derived from Kaplan-Meier estimation for the range of recorded survival times, including 95% Confidence Interval."
x |> 
  gt() |> 
  gt::fmt_percent(columns = c(surv, upper, lower)) |> 
  gt::fmt_number(columns = c(std.err), decimals = 2)
```

```{r}
#| label: tbl-dict
#| tbl-cap: Data dictionary of the final data set used for the analysis. This combines the case reports and supplied data for available and references studies. The data has been transformed into a tidy format with formatting applied. 
tar_load(subjects)
make_data_dictionary(subjects)
```

\newpage

# References {.unnumbered}
